### ISSUE TITLE
Safari is not allow you to edit a text Versión 12.1.1 (14607.2.6.1.1)

### ISSUE THREAD

**Issue by jesusdp** [artf/grapesjs#2218]
If I try to edit an element on the container is not appearing to be working or if I decide to be bold nothing at all.

Chrome, IE are working fine only on Safari is the issue.

This is my current settings:

startGrapeJS : function() {
    var _this = this;
    
    _this.editor = grapesjs.init({
      // Indicate where to init the editor. You can also pass an HTMLElement
      container: '#gjs',
      // Get the content for the canvas directly from the element
      // As an alternative we could use: `components: '<h1>Hello World Component!</h1>'`,
      fromElement: true,
      // Size of the editor
      height: '300px',
      width: 'auto',
      // Disable the storage manager for the moment
      storageManager: false,
      // Avoid any default panel
      panels: { defaults: [] },
      assetManager: {
        assets: [
          'https://designerpages.s3.amazonaws.com/assets/73040011/5ddb8db6-1034-4f95-a78c-b21c1de7833d.jpg',
          'https://designerpages.s3.amazonaws.com/assets/53571422/Deft_0_a.jpg'
        ],
        upload: false,
      },
      blockManager: {
        appendTo: '#blocks',
        blocks: [
          { 
            id: 'section', // id is mandatory
            label: '<b>Section</b>', // You can use HTML/SVG inside labels
            attributes: { class:'gjs-block-section' },
            content: '<section>' +
              '<h1>This is a simple title</h1>' +
              '<div>This is just a Lorem text: Lorem ipsum dolor sit amet</div>' +
              '</section>',
          },
          { 
            id: 'text',
            label: 'Text',
            content: '<div data-gjs-type="text">Insert your text here</div>',
          },
          { 
            id: 'image',
            label: 'Image',
            // Select the component once it's dropped
            select: true,
            // You can pass components as a JSON instead of a simple HTML string,
            // in this case we also use a defined component type `image`
            content: { type: 'image' },
            // This triggers `active` event on dropped components and the `image`
            // reacts by opening the AssetManager
            activate: true,
          }
        ]
      },
    });
}

**Comment by artf** [comment]
Duplicate of #2210
